fixtures/validation/fixture-1171.yaml

```diff
@@ -58,10 +58,7 @@ paths:
       responses:
         '200':
           schema:
-            # $ref may not have sibling
-            properties:
```

Comment:
We are testing a success case in this fixture. This part is an invalid swagger spec. The go-openapi/spec improvements validation now catches this. Removing this to keep the testing logic that expects a valid spec.